dbt/include/databricks/macros/materializations/incremental/incremental.sql

```diff
@@ -94,16 +98,19 @@
     {%- endif -%}
     {% do apply_liquid_clustered_cols(target_relation) %}
     {% if _configuration_changes is not none %}
-      {% set tags = _configuration_changes.changes["tags"] %}
+      {% set tags = _configuration_changes.changes.get("tags", None) %}
```

Comment:
Now that we have two possible changes, we need to be able to check for presence of one or the other in the change set.